charts/ambassador/CHANGELOG.md

```diff
@@ -13,6 +14,10 @@ numbering uses [semantic versioning](http://semver.org).
 
 - Update Ambassador to version 1.12.4: [CHANGELOG](https://github.com/datawire/ambassador/blob/master/CHANGELOG.md)
 
+## v6.7.0
```

Comment:
is 6.7.0 supposed to come after 6.6.4?
also i think the things in `Next Release` category belong under 6.7.0?